airbyte/caches/base.py

```diff
@@ -74,6 +74,11 @@ def get_database_name(self) -> str:
         """Return the name of the database."""
         ...
 
+    def get_database_connection_via_alternate_method(self) -> object:
```

Comment:
i could not think of a better way to do this :/ we need to use Snowflake Python connector for checking destination schema.

Reply:
Maybe just a rename of the method, perhaps something like 

`get_vendor_client()` or `get_custom_client()`?

Reply:
Updated name.